.env.development

```diff
@@ -0,0 +1,8 @@
+HANDLE_TARGET_HOSTNAME=localhost:3000
+HANDLE_EMAIL=contact@example.com
+HANDLE_PREFIX=1234.DEV
+HANDLE_ADMIN_PREFIX='"1234.TEST/ADMIN"'
```

Comment:
It looks like the `ACTION_MAILER_*` environment variables got removed from the file during the rebase.